Source/core/resources/fuels.json

```diff
@@ -0,0 +1,58 @@
+{  
+  "Diesel":{  
+    "langKey":"fuelDiesel",
+    "measuredBy":"v",
+    "CO2Emission":"2614"
+  },
+  "Gasoline":{  
+    "langKey":"fuelGasoline",
+    "measuredBy":"v",
+    "CO2Emission":"2328"
+  },
+  "E10":{  
+    "langKey":"fuelE10",
+    "measuredBy":"v",
+    "CO2Emission":"2245"
+  },
+  "E25":{  
+    "fuel":"E25",
+    "langKey":"fuelE25",
+    "measuredBy":"v",
+    "CO2Emission":"2121"
+  },
+  "E85":{  
+    "langKey":"fuelE85",
+    "measuredBy":"v",
+    "CO2Emission":"1626"
+  },
+  "Ethanol":{  
+    "langKey":"fuelEthanol",
+    "measuredBy":"v",
+    "CO2Emission":"1503"
+  },
+  "Bio-Diesel":{  
+    "langKey":"fuelBioDiesel",
+    "measuredBy":"v",
+    "CO2Emission":"2486"
+  },
+  "B20":{  
+    "langKey":"fuelB20",
+    "measuredBy":"v",
+    "CO2Emission":"2588"
+  },
+  "LPG":{  
+    "langKey":"fuelLPG",
+    "measuredBy":"m",
+    "CO2Emission":"1533"
+  },
+  "CNG":{  
```

Comment:
Stands for [Compressed Natural Gas](https://en.wikipedia.org/wiki/Compressed_natural_gas). Composed primarily of Methane in contrast to LPG which chiefly consists of propane, propylene, butane, and butylene